anxcloud/testutils/recorder/preifx_recorder.go

```diff
@@ -0,0 +1,47 @@
+package recorder
+
+import (
+	"context"
+	"errors"
+	"go.anx.io/go-anxcloud/pkg/client"
+	"go.anx.io/go-anxcloud/pkg/ipam/prefix"
+)
+
+type prefixCleanupHandler func(ctx context.Context) error
+
+type PrefixRecorder struct {
+	handler []prefixCleanupHandler
+}
+
+func (p PrefixRecorder) Cleanup(ctx context.Context) []error {
+	var cleanUpErrors []error
+	for _, handler := range p.handler {
+		err := handler(ctx)
+		if err != nil {
+			cleanUpErrors = append(cleanUpErrors, err)
+		}
+	}
+	return cleanUpErrors
+}
+
+func (p *PrefixRecorder) RecordPrefixByID(identifier string) {
+	p.handler = append(p.handler, func(ctx context.Context) error {
```

Comment:
Reading this anon functions body confused me, as I didn't read it being an anon function and it clearly didn't "Record the Prefix by ID" - maybe make it a normal function, but up to you - both ways have pros and cons

Reply:
I decided to get rid of it completely for now. Because

- It was not used. 
- Recording the prefix by ID is rather difficult in the terraform test suits
* recording them by name is even more difficult since it changes depending on the state
* we don't have a problem with this test leaking prefixes. (the problem of leaking resources occurs mainly on VMs and other resources that had problem. This